e2e/fn_test.go

```diff
@@ -43,7 +43,9 @@ func runTests(t *testing.T, path string) {
 	for _, c := range *cases {
 		c := c // capture range variable
 		t.Run(c.Path, func(t *testing.T) {
-			t.Parallel()
```

Comment:
To make things clear and predictable, can we run the parallel test cases and sequential test cases separately? Also please update the test harness readme.

Reply:
Done